R/derive_extreme_records.R

```diff
@@ -159,6 +168,7 @@
 #'   order = exprs(desc(AVAL), AVISITN),
 #'   mode = "first",
 #'   filter_add = !is.na(AVAL),
+#'   keep_vars_source = ,
```

Comment:
I think this looks a little strange, see above as well

Reply:
Could you give more clarity on this?

Reply:
Got it, this is updated now!